mappings/net/minecraft/entity/ai/pathing/EntityNavigation.mapping

```diff
@@ -40,6 +40,7 @@ CLASS net/minecraft/class_1408 net/minecraft/entity/ai/pathing/EntityNavigation
 	METHOD method_26085 resetNode ()V
 	METHOD method_31266 resetNodeAndStop ()V
 	METHOD method_31267 isNearPathStartPos ()Z
+	METHOD method_35143 getNodeReachProximity ()F
```

Comment:
Hmm, I forgot if we mostly use range or proximity. Might be good to unify them (later, not now)